packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1803,6 +1803,9 @@ class EditableText extends StatefulWidget {
   ///   Autofill service. Enable the autofill service of your choice, and make
   ///   sure there are available credentials associated with your app.
   ///
+  /// Using [InputDecoration.hintText] can help password manager like Samsung Pass to well
```

Comment:
uber nit: maybe: "Specifying [InputDecoration.hintText] may also help autofill services determine the expected content type of an input field, although this is typically not required when `autofillHints` are present."

Reply:
Done, changed to put have 

```dart
/// Specifying [InputDecoration.hintText] may also help autofill services
/// (like Samsung Pass) determine the expected content type of an input field,
/// although this is typically not required when autofillHints are present.
 ```
 
 I've added Samsung Pass, i believe this can help developper which face issue related to Samsung.